usaspending_api/awards/v2/filters/award.py

```diff
@@ -149,11 +148,9 @@ def award_filter(filters):
 
         # place_of_performance
         elif key == "place_of_performance_locations":
-            or_queryset = []
-            for v in value:
-                or_queryset.append(v)
-            if len(or_queryset) != 0:
-                queryset &= Award.objects.filter(place_of_performance__location_id__in=or_queryset)
+            or_queryset = geocode_filter_locations('place_of_performance', value, 'award')
```

Comment:
Seeing the calls of this function in the `transaction_filter` use `TransactionNormalized`, I'm surprised `apps.get_model('awards', model)` works just as fine with `award` (not `Award`). Minuscule but for clarification can you capitalize these to make it clear we're dealing with the `Award` table?

Reply:
Fixed to Award